src/components/ui/MasqAlert.jsx

```diff
@@ -0,0 +1,62 @@
+/* global _ */
+import React, { useState, useEffect } from 'react';
+import nconf from '@qwant/nconf-getter';
+import Alert from 'src/components/ui/Alert';
+
+const masqAlertEnabled = nconf.get().masq.alertEnabled;
+const masqLink = nconf.get().masq.link;
+
+const MasqAlert = () => {
+  if (!masqAlertEnabled) {
+    return null;
+  }
+
+  let masqDismissed = false;
+  try {
+    masqDismissed = window.localStorage.getItem('masq_alert_dismissed');
+  } catch (e) {
+    console.error(e);
+  }
+
+  const [masqAlertDate, setMasqAlertDate] = useState('');
+  const [isVisible, setIsVisible] = useState(masqAlertEnabled && masqDismissed !== 'true');
+  if (!isVisible) {
+    return null;
+  }
+
+  useEffect(() => {
+    setMasqAlertDate(Intl
+      .DateTimeFormat(window.getLang().locale.replace('_', '-'))
```

Comment:
The default format uses only numeric values, and it may be unclear for users whether the GB or US format is used here.

```suggestion
      .DateTimeFormat(window.getLang().locale.replace('_', '-'), {day: "numeric", month: "long", year: "numeric"})
```